shared/stories/__tests__/__snapshots__/Storyshots.test.js.snap

```diff
@@ -8042,13 +8042,13 @@ Array [
                                   onClick={[Function]}
                                   onMouseEnter={[Function]}
                                   onMouseLeave={[Function]}
-                                  title="flag-vu"
+                                  title="baby"
```

Comment:
hmm why did storyshots change?

Reply:
The reactions on messages are “randomly” generated. So, the new random ones are in the snapshot.